internal/db/db.go

```diff
@@ -533,3 +548,30 @@ func printStore(ctx context.Context, store corekv.ReaderWriter) error {
 
 	return iter.Close()
 }
+
+func (db *DB) GetSearchableEncryptionKey() []byte {
+	return db.searchableEncryptionKey
+}
+
+// QueryDocIDsByValues queries SE artifacts from replicators based on field values.
+func (db *DB) QueryDocIDsByValues(
```

Comment:
todo: I think this function name needs to reflect that it is use SE.  At the moment the name is very generic.